### DISCUSSION TITLE
Conditional render

### DISCUSSION THREAD

**Discussion by lapenji** [alexzaganelli/strapi-plugin-email-designer#145] [Q&A] [answered]
Hello and thank you for your precious work!

I was wondering if is possible to conditional render something in an email template, i have something like this:

`Nome: {{name}}

Company: {{company}}

Applicazioni: {{applications}}

Telefono: {{phone}}`

I would like to render Company: only if company is not null or undefined, is it possible?

Thank you in advance!

**Comment by creazy231** [comment, marked as answer]
related to this post on https://stackoverflow.com/questions/8548943/how-to-handle-an-if-statement-in-a-mustache-template it should be possible using

```
{{#value}}
  value is true
{{/value}}
{{^value}}
  value is false
{{/value}}
```

**Reply by lapenji** [reply]
Thanks a lot it works perfectly! 
I did not know that it was called mustache template, next time I will search better by myself!

Thanks again
